### ISSUE TITLE
Firefox Nightly: Loading failed for the <script> with source...

### ISSUE THREAD

**Issue by rwaldron** [jsbin/jsbin#3219]
Console shows these errors: 

```
Loading failed for the <script> with source “https://static.jsbin.com/js/vendor/jquery-1.11.0.min.js”.
Loading failed for the <script> with source “https://static.jsbin.com/js/prod/jsbin-4.1.2.min.js”.
```

I can't figure out why they aren't loading, but I'll keep looking.

**Comment by scheinercc** [comment]
Just filed a bug in bugzilla against it: https://bugzilla.mozilla.org/show_bug.cgi?id=1444687. Will link to this page from there as well.

**Comment by scheinercc** [comment]
Apparently a Symantec cert is in use here and the distrust (Firefox/Chrome) of these certs has started: see reference bug https://bugzilla.mozilla.org/show_bug.cgi?id=1434300

**Comment by remy** [comment]
Except i updated the certs…

**Comment by scheinercc** [comment]
@remy quick turn around! then I'd suggest you reopen and comment on https://bugzilla.mozilla.org/show_bug.cgi?id=1444687

**Comment by remy** [comment]
No, I update the cert 3 weeks ago. *that's* the problem. You (and many
others) are somehow seeing a cached cert.

**Comment by scheinercc** [comment]
Sorry, I am not really into the whole cert thing. Just follow along the news. 
Wouldn't the caching be something with the browser? And therefore part of the bug I linked?

**Comment by scheinercc** [comment]
remy, it's still blocked for me. Checked in Nightly and DevEdition with VPN set to Europe and US. Will try again later.

**Comment by rwaldron** [comment]
Still having the issue here.

**Comment by scheinercc** [comment]
It's back :) Thank you!

**Comment by remy** [comment]
Finally solved it last night. Turned out i had a damn etc/hosts entry that was confusing me!
